cmd/flow-dps-indexer/main.go

```diff
@@ -206,6 +211,26 @@ func run() int {
 		log.Info().Time("finish", finish).Str("duration", duration.Round(time.Second).String()).Msg("Flow DPS Indexer stopped")
 	}()
 
+	// Expose badgerDB and go metrics.
+	go func() {
+		if flagMetrics == "" {
+			return
+		}
+
+		start := time.Now()
+		log.Info().Time("start", start).Msg("Metrics server starting")
```

Comment:
Current logging uses lower case for all messages, so `metrics` will do just fine.